src/test/fuzz/chain.cpp

```diff
@@ -14,28 +14,32 @@
 FUZZ_TARGET(chain)
 {
     FuzzedDataProvider fuzzed_data_provider(buffer.data(), buffer.size());
-    std::optional<CDiskBlockIndex> disk_block_index = ConsumeDeserializable<CDiskBlockIndex>(fuzzed_data_provider);
-    if (!disk_block_index) {
+    const std::vector<uint8_t> index_buff = ConsumeRandomLengthByteVector(fuzzed_data_provider);
```

Comment:
why?

If there is a reason for the change, it would be good to make it in a separate commit, or explain why it needs to be done in this commit.

If there is no reason, it would be better to not make the change.

Reply:
Huh, weird, can't remember why I did this but as you suggest it does compile without. Removed.